google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -1366,6 +1367,22 @@ final class PooledSession implements Session {
     private volatile SpannerException lastException;
     private volatile boolean allowReplacing = true;
 
+    /**
+     * Property to mark if the session is eligible to be long-running. This can only be true if the
```

Comment:
Can we add here what long running is, as in the 60 minute limit?

Reply:
* I was avoiding adding the exact configuration value into docs since it may change in future and i'd like to avoid the docs getting stale. 
* That said, in the documentation of `SessionPool#PoolMaintainer` class we have explicitly given some examples where we have mentioned 60 minutes.

Let me know if the readability sounds ok.